deploy.sh

```diff
@@ -6,7 +6,7 @@ then
     export KUBE_TOKEN=${PROD_KUBE_TOKEN}
     export DNS_PREFIX=
 else
-    export DNS_PREFIX=${ENVIRONMENT}.
+    export DNS_PREFIX=${ENVIRONMENT}.notprod.
```

Comment:
so production url will be:
https://fm.ip.pttg.homeoffice.gov.uk
test:
https://test.notprod.fm.ip.pttg.homeoffice.gov.uk
dev:
https://dev.notprod.fm.ip.pttg.homeoffice.gov.uk